kernel/src/transaction/mod.rs

```diff
@@ -478,6 +498,23 @@ pub enum CommitResult {
     Conflict(Transaction, Version),
 }
 
+/// Generate an In-Commit Timestamp (ICT) following the protocol requirement.
```

Comment:
"following the protocol requirement" isn't very useful to say, lets just document a bit more directly